### ISSUE TITLE
since version 2.0.878 enum is incorrected

### ISSUE THREAD

**Issue by dlasvazi** [mganss/XmlSchemaClassGenerator#421]
Version 2.0.878
/// <summary>
        /// <para>Исключительные права на результат интеллектуальной деятельности</para>
        /// </summary>
        [System.ComponentModel.DescriptionAttribute("Исключительные права на результат интеллектуальной деятельности")]
        [System.Xml.Serialization.XmlEnumAttribute("11")]
        Item11,
        
        /// <summary>
        /// <para>Произведение науки, литературы или искусства</para>
        /// </summary>
        [System.ComponentModel.DescriptionAttribute("Произведение науки, литературы или искусства")]
        [System.Xml.Serialization.XmlEnumAttribute("11.1")]
        Item11Period1,

Version > 2.0.878
/// <summary>
        /// <para>Исключительные права на результат интеллектуальной деятельности</para>
        /// </summary>
        [System.ComponentModel.DescriptionAttribute("Исключительные права на результат интеллектуальной деятельности")]
        [System.Xml.Serialization.XmlEnumAttribute("11")]
        Item111,
        
        /// <summary>
        /// <para>Произведение науки, литературы или искусства</para>
        /// </summary>
        [System.ComponentModel.DescriptionAttribute("Произведение науки, литературы или искусства")]
        [System.Xml.Serialization.XmlEnumAttribute("11.1")]
        Item111,

enum with xml attribute 11 and 11.1 was value Item111

xsd
<xs:enumeration value="11">
				<xs:annotation>
					<xs:documentation>Исключительные права на результат интеллектуальной деятельности</xs:documentation>
				</xs:annotation>
			</xs:enumeration>
			<xs:enumeration value="11.1">
				<xs:annotation>
					<xs:documentation>Произведение науки, литературы или искусства</xs:documentation>
				</xs:annotation>
			</xs:enumeration>

**Comment by mganss** [comment]
I think this is due to #413. Sorry for the confusion. This is just a naming issue, not a functional one, right?

**Comment by dlasvazi** [comment]
Yeah! Value '11' converted into Item111 enum name and Value '11.1' is converted into Item111 enum
and there is conflict in c# code

**Comment by mganss** [comment]
Using the following as the whole XSD I'm unable to repro:

```xsd
<xs:simpleType name="EnumTest">
<xs:restriction base="xs:string">
	<xs:enumeration value="11">
		<xs:annotation>
			<xs:documentation>Исключительные права на результат интеллектуальной деятельности</xs:documentation>
		</xs:annotation>
	</xs:enumeration>
	<xs:enumeration value="11.1">
		<xs:annotation>
			<xs:documentation>Произведение науки, литературы или искусства</xs:documentation>
		</xs:annotation>
	</xs:enumeration>
</xs:restriction>
</xs:simpleType>
```

I suspect that in your XSD there is another enum value that gets the name `Item11` which causes the enum member for "11" to get a "1" added to the end which in turn conflicts with `Item111` for "11.1". Can you share all of the enum members in your XSD?
